docs/CustomRoutes.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: default
+title: "The CustomRoutes Component"
+---
+
+# `<CustomRoutes>`
+
+To register your own routes, pass one or several `<CustomRoutes>` elements as children of `<Admin>`. Declare as many [react-router-dom](https://reactrouter.com/docs/en/v6/api#routes-and-route) `<Route>` as you want inside them:
+
+```jsx
+// in src/App.js
+import * as React from "react";
+import { Admin, Resource, CustomRoutes } from 'react-admin';
+import posts from './posts';
+import comments from './comments';
+import Foo from './foo';
+import Bar from './bar';
+
+const App = () => (
+    <Admin dataProvider={simpleRestProvider('http://path.to.my.api')}>
+        <Resource name="posts" {...posts} />
+        <Resource name="comments" {...comments} />
+        <CustomRoutes>
+            <Route path="/foo" element={<Foo />} />
+            <Route path="/bar" element={<Bar />} />
```

Comment:
I suggest we stop using foo/bar in our examples. First, it's very far from real usages. Second the story behind those is horrible
```suggestion
            <Route path="/settings" element={<Settings />} />
            <Route path="/profile" element={<Profile />} />
```